.github/workflows/load-test.yml

```
@@ -0,0 +1,108 @@
+name: Load Test
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
+  workflow_dispatch:
+    inputs:
+      test_id:
```

```suggestion
      test-id:
```
I guess this isn't a lint rule, but all GH Actions ones use dashes instead of underscores.